README.md

```diff
@@ -79,7 +78,7 @@ From there, you can use the **dev** binary to manage the environment.
 
 ## Save password
 
-To avoid having to enter the SUDO password all the time we will encrypt our SUDO password  with a password that will be stored in the keychain :
+To avoid having to enter the SUDO password all the time we will encrypt our SUDO password with a password that will be stored in the keychain :
```

Comment:
```suggestion
To avoid having to enter the SUDO password all the time we will encrypt our SUDO password with a password that will be stored in the keychain:
```